deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/StakeDeeds.vue

```diff
@@ -18,10 +18,10 @@
 -->
 <template>
   <v-card flat>
-    <v-card-title class="ps-0">
+    <v-card-title class="ps-0 headline font-weight-medium">
```

Comment:
It's may be better to define generic CSS for `.v-card-title` to use a different font-size in all dapp.

Reply:
done!